tracer/README.MD

```diff
@@ -0,0 +1,21 @@
+﻿# dd-trace-dotnet/tracer
+
+This repo ([dd-trace-dotnet](https://github.com/DataDog/dd-trace-dotnet)) contains the sources for the client-side components of the Datadog product suite for Application Telemetry Collection and Application Performance Monitoring for .NET Applications.
+
+This directory (`/tracer`) contains the source for the __Tracer for .NET__ product, which is a part of that product suite.
+
+### Copyright
+
+Copyright (c) 2017-2021 Datadog
```

Comment:
```suggestion
Copyright (c) 2017 Datadog
```

Reply:
Addressed in https://github.com/DataDog/dd-trace-dotnet/commit/8b9642541f0ad21873f3bdd9325b6601a10bb8c9